http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1497,14 +1497,14 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorReport"
-        "420":
-          description: App not ready
+        "422":
+          description: App is stopped
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorReport"
-        "422":
-          description: App is stopped
+        "423":
+          description: App not ready
```

Comment:
What was the reason for the `42*` response code changes?

Reply:
I just noticed that the status code returned by Leo didn't match the status code in swagger, so I updated Swagger docs.